kavm/src/kavm/kavm.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import re
+import subprocess
+from pathlib import Path
+from subprocess import CompletedProcess
+from typing import Any, Final, Iterable, List, Optional, Union
+
+from pyk.cli_utils import run_process
+from pyk.kast import KSort
+from pyk.ktool import KRun
+from pyk.prelude import Sorts
+
+_LOGGER: Final = logging.getLogger(__name__)
+
+
+class KAVM(KRun):
+    """
+    Interact with the K semantics of AVM: evaluate Algorand transaction groups
+    """
+
+    def __init__(
+        self,
+        definition_dir: Path,
+        use_directory: Any = None,
+        logger: Optional[logging.Logger] = None,
```

Comment:
Consider importing `Logger`.

```suggestion
        logger: Optional[Logger] = None,
```